scripts/cibuild

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${COVID19_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Generate the site!
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        jekyll build
+
+        mkdir -p "${DIR}/../_site/maps"
+
+        for map in "${DIR}"/../maps/*; do
```

Comment:
This is what handles digging into each of the map subdirectories to execute each map specific `cibuild` script.